### PR TITLE
Add environment variables to all cli flags

### LINKED ISSUE #62: Enable configuration using environment variables
@derlin introduced a new configuration parameter (for aws signing) on the latest release. It can be configured either by an environment variable or by flag in the `cli`.

It would be nice to have the other parameters to be configurable by environment variables.
It must have the same prefix @derlin used for the aws signing feature. 

Thanks!

### PR DESCRIPTION
**Pull request by derlin**
Closes #62

### CODE REVIEW

**Comment by derlin** [comment]
@angelbarrera92  I have a doubt for the prometheus endpoints. In a way `PROM_PROXY_PROMETHEUS_ENDPOINT` is quite long, but on the other it stays coherent. Let me know what you think!

**Comment by angelbarrera92** [comment]
> @angelbarrera92 I have a doubt for the prometheus endpoints. In a way `PROM_PROXY_PROMETHEUS_ENDPOINT` is quite long, but on the other it stays coherent. Let me know what you think!

Hi @derlin!

Yeee, i know. It looks "too long", on the other side, it make sense to me to have it as it is, 

Dont worry that much

File: README.md [hunk 1]
```diff
@@ -33,19 +33,27 @@ instance, configure the auth proxy configuration and run it.
...
+Available arguments // environment variables to the `run` command:
 
-- `--port`: Port used to expose this proxy.
-- `--prometheus-endpoint`: URL of your Prometheus instance.
-- `--reload-interval`: Interval in minutes to reload the auth config file.
-- `--unprotected-endpoints`: Comma separated list of endpoints that do not require authentication.
-- `--auth-type`: Type of authentication to use, one of `basic`,  `jwt`
-- `--auth-config`: Authentication configuration.
+- `--port` // `PROM_PROXY_PORT`: Port used to expose this proxy.
+- `--prometheus-endpoint` // `PROM_PROXY_ENDPOINT`: URL of your Prometheus instance.
```
Comment (angelbarrera92) [inline README.md:47]: Lets set `PROM_PROXY_PROMETHEUS_ENDPOINT`

**Review by angelbarrera92 — APPROVED** [review]
LGTM!

**Pull request merged**